enterprise/server/cmd/ci_runner/main.go

```diff
@@ -99,15 +100,28 @@ var (
 	fallbackToCleanCheckout = flag.Bool("fallback_to_clean_checkout", true, "Fallback to cloning the repo from scratch if sync fails (for testing purposes only).")
 
 	shellCharsRequiringQuote = regexp.MustCompile(`[^\w@%+=:,./-]`)
-
-	// initLogs contain informational logs from the setup phase (cloning the
-	// git repo and deciding which actions to run) which are reported as part of
-	// the first action's logs.
-	initLog bytes.Buffer
 )
 
+type workspace struct {
+	// Whether the pushed branch could not be cleanly merged into the targt
+	// branch.
+	mergeConflict bool
```

Comment:
ubernit: flip to mergesCleanly ("positive" variables are easier to read and grok)

Then you can have if !mergesCleanly (if not merges cleanly)?